src/analysis/retail/shaman/restoration/modules/Abilities.ts

```diff
@@ -342,7 +342,7 @@ class Abilities extends CoreAbilities {
         },
       },
       {
-        spell: TALENTS.LAVA_BURST_TALENT.id,
+        spell: SPELLS.LAVA_BURST.id,
```

Comment:
```suggestion
        spell: TALENTS.LAVA_BURST_TALENT.id,
```
this was already correct as the talent version

Reply:
I removed the changes I made for resto sham since this PR is focused on ele shaman and I wanted to avoid confusion. I can add back in the edits you are requesting though